docs/api/api-specification.md

```diff
@@ -0,0 +1,2170 @@
+                  "display": "English"
+                }
+              ]
+            }
+          }
+        ],
+        "resourceType": "Patient"
+      }
+    },
+    {
+      "fullUrl": "http://localhost:3000/fhir/r4/QuestionnaireResponse/3",
+      "resource": {
+        "id": 3,
+        "meta": {
+          "lastUpdated": "2020-05-29T00:42:54+00:00"
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/3"
+        },
+        "item": [
+          {
+            "linkId": "0",
+            "text": "cough",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "1",
+            "text": "difficulty-breathing",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "2",
+            "text": "fever",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "3",
+            "text": "used-a-fever-reducer",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "4",
+            "text": "chills",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "5",
+            "text": "repeated-shaking-with-chills",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "6",
+            "text": "muscle-pain",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "7",
+            "text": "headache",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "8",
+            "text": "sore-throat",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "9",
+            "text": "new-loss-of-taste-or-smell",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          }
+        ],
+        "resourceType": "QuestionnaireResponse"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+
+<a name="create"/>
+
+### Creating
+
+The API supports creating new monitorees.
+
+#### Extensions
+
+<a name="create-ext"/>
+
+Along with supporting the US Core extensions for race, ethnicity, and birthsex, Sara Alert includes four additional extensions for things specific to the Sara Alert workflows.
```

Comment:
Should we link to these extensions here?

Reply:
Done https://github.com/SaraAlert/SaraAlert/pull/598/commits/490bef24961bf80cb728c31cd3185f0d063a4d0b